elements/pouch-db/demo/index.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+    <title>PouchDb: pouch-db Demo</title>
+    <script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+    <script src="../node_modules/@lrnwebcomponents/deduping-fix/deduping-fix.js"></script>
+    <script src="../node_modules/web-animations-js/web-animations-next-lite.min.js"></script>
+    <script type="module">
+      import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
+      import '@polymer/iron-demo-helpers/demo-snippet.js';
+      import '@lrnwebcomponents/multiple-choice/multiple-choice.js';
+      import '../pouch-db.js';
+    </script>
+    <style is="custom-style" include="demo-pages-shared-styles"></style>
+  </head>
+  <body>
+    <div class="vertical-section-container centered">
+      <h3>Basic pouch-db demo</h3>
+      <demo-snippet>
+        <template>
+          <pouch-db>
+            This is pouch-db
+          </pouch-db>
+
+          <multiple-choice quiz-name="Quiz1"></multiple-choice>
```

Comment:
Here's 2 legit multiple choice elements from the haxtheweb.org site so you can do some simple sending of correct / incorrect
```html
<multiple-choice style="width: 75%; margin: 0px auto; display: block;" check-label="Check guess" reset-label="Clear" hide-title question="Who founded the ELMS:LN Platform?" correct-text="Great job!" incorrect-text="Try again" randomize typeof="oer:Assessment" resource="#97ffed48-c928-39ae-1bea" prefix="oer:http://oerschema.org/ schema:http://schema.org/ dc:http://purl.org/dc/terms/ foaf:http://xmlns.com/foaf/0.1/ cc:http://creativecommons.org/ns# bib:http://bib.schema.org " answers="[{&quot;correct&quot;:false,&quot;label&quot;:&quot;Michael Potter&quot;},{&quot;correct&quot;:true,&quot;label&quot;:&quot;Bryan Ollendyke&quot;},{&quot;correct&quot;:false,&quot;label&quot;:&quot;Mythic Elves&quot;},{&quot;correct&quot;:false,&quot;label&quot;:&quot;Charles Lavera&quot;},{&quot;correct&quot;:false,&quot;label&quot;:&quot;Nikki Massaro Kauffman&quot;}]" schema-resource-id="#97ffed48-c928-39ae-1bea"></multiple-choice>
<multiple-choice style="width: 75%; margin: 0px auto; display: block;" title="ELMS Development team" check-label="Check your guess" reset-label="Clear answers" question="Who is responsible for ELMS:LN existing? Who improves it and wants it to be the best NGDLE solution to transform education and the world?" correct-text="Great job! You got our trick question!" incorrect-text="Try again, there's some more answers..." randomize typeof="oer:Assessment" resource="#59426af4-ecf2-f49d-06a6" prefix="oer:http://oerschema.org/ schema:http://schema.org/ dc:http://purl.org/dc/terms/ foaf:http://xmlns.com/foaf/0.1/ cc:http://creativecommons.org/ns# bib:http://bib.schema.org " answers="[{&quot;correct&quot;:true,&quot;label&quot;:&quot;Bryan Ollendyke&quot;},{&quot;correct&quot;:true,&quot;label&quot;:&quot;Charles Lavera&quot;},{&quot;correct&quot;:true,&quot;label&quot;:&quot;Nikki Massaro Kauffman&quot;},{&quot;correct&quot;:true,&quot;label&quot;:&quot;Anyone, ELMS:LN is an open community developed by multiple organizations!&quot;},{&quot;correct&quot;:true,&quot;label&quot;:&quot;Michael Potter&quot;}]" schema-resource-id="#59426af4-ecf2-f49d-06a6"></multiple-choice>
```

Reply:
Got em.  thanks.